src/frontend/screens/ObservationEdit/ObservationEditView.js

```diff
@@ -24,7 +26,17 @@ const LocationView = ({ longitude, latitude, accuracy }) => (
       <Text>Searching...</Text>
     ) : (
       <>
-        <FormattedCoords lat={latitude} lon={longitude} />
+        <MaterialIcons
+          size={14}
+          name="location-on"
+          color="orange"
+          style={{ marginRight: 5 }}
+        />
+        <FormattedCoords
+          style={styles.locationText}
+          lat={latitude}
+          lon={longitude}
+        />
         {accuracy && (
           <Text style={styles.accuracy}>{"±" + accuracy.toFixed(2) + "m"}</Text>
```

Comment:
```suggestion
          <Text style={styles.accuracy}>{" ±" + accuracy.toFixed(2) + "m"}</Text>
```

This needs an extra space in the new design so it isn't too close to the coordinates